.gitignore

```diff
@@ -11,3 +11,4 @@ pyproject.toml
 .cache/
 .pytest-cache/
 .pytest_cache/
+.idea/
```

Comment:
this has nothing to do with this contribution.
could you please remove this change?